### PR TITLE
Create calculator template

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this project, I continued with the development of the Math Magicians app. I developed a React component that will hold the core functionality: a calculator. For this purpose, I followed these steps:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,7 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <Calculator />;
```
Comment (mricanho) [inline src/App.js:4]: - [ ] This component should be class-based. e.g.` class App extends Component {}`

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            value={textItem}
+            className={`button grid-item ${className || ''}`}
+            key={textItem}
+            type="button"
+          />
+        ))}
+      </div>
+    </div>
+  );
+};
```
Comment (mricanho) [inline src/components/Calculator.js:87]: - [ ] This component should be class-based. e.g.

imagen

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @tacettinkustu, 

Good job so far! Your Gitflow, commit messages and style are corrects!🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Fix your README file and follow the tempĺate from microverse, remember that this is a good practice.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

believe in change

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tacettinkustu** [comment]
> Hi @tacettinkustu,
> 
> Good job so far! Your Gitflow, commit messages and style are corrects!🔥🔥
> 
> There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> * [x]  Fix your README file and follow the tempĺate from microverse, remember that this is a good practice.
> 
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> believe in change
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

README file will be created later according to project phases.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @tacettinkustu 🔥 

You worked hard in all the instructions that I previously give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**